patches/minecraft/net/minecraft/entity/EntityLivingBase.java.patch

```diff
@@ -247,15 +263,63 @@
                      {
                          d1 = d11;
                          d13 = this.field_70163_u + 2.0D;
-@@ -1781,6 +1807,7 @@
+@@ -1781,13 +1809,14 @@
          }
  
          this.field_70160_al = true;
 +        net.minecraftforge.common.ForgeHooks.onLivingJump(this);
      }
  
      protected void func_70629_bd()
-@@ -1874,7 +1901,8 @@
+     {
+-        this.field_70181_x += 0.03999999910593033D;
++        this.field_70181_x += 0.03999999910593033D * this.func_110148_a(SWIM_SPEED).func_111126_e();
+     }
+-
++    
+     protected void func_180466_bG()
+     {
+         this.field_70181_x += 0.03999999910593033D;
+@@ -1797,7 +1826,38 @@
+     {
+         return 0.8F;
+     }
++    
++    public void func_191958_b(float strafe, float up, float forward, float friction)
```

Comment:
please add an `@Override` annotation and move this to the end of the file after a comment like this:
`/* ======================================== FORGE START ======================================== */`

Reply:
When I moved from EntityPlayer to EntityLivingBase I didn't realize there was a Forge section already. I have moved the overridden method there and added the annotation.